src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/missingleadingasterisk/InputMissingLeadingAsterisk.java

```diff
@@ -0,0 +1,20 @@
+package com.puppycrawl.tools.checkstyle.checks.javadoc.missingleadingasterisk;
+
+public class InputMissingLeadingAsterisk {
+  /**
+   ** OK
+   *
+       violation
+   violation
+
+   **/
+  public void test(int a) {}
+
+  /** This is an inline comment */
+  public void test2() {}
+
+  /*
+      This is a non javadoc comment.
+   */
+  /**/ // This is empty comment for 100% code coverage.
```

Comment:
`// This is empty comment for 100% code coverage.` please remove this comment and add one more example - empty single line javadoc `/***/`

Reply:
Done